cmd/asset/main.go

```diff
@@ -111,7 +112,10 @@ func main() {
 			logger.WithError(err).Fatal("Cannot parse standalone config")
 		}
 
-		rootRouter = server.NewRouter()
+		router = server.NewRouter()
+		router.HandleFunc("/_healthz", server.HealthCheckHandler)
```

Comment:
Actually the router returned by server.NewRouter does have a health check endpoint at `/healthz`. Maybe also add `/_healthz` in `server.NewRouter()`?

Reply:
Oh I didn't know there is `/healthz`, let me update the deployment to use `/healthz` then

Reply:
Oh, I found the problem is we add the middleware to the `rootRouter`, so the `/healthz` pass through those middleware but it should not. 🤔
Or we should not add the `/healthz` in `NewRouter` and let the gear to control the whole route? @louischan-oursky

Reply:
Then NewRouter is just a 3-line function. Maybe remove it entirely.